bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -48,30 +56,129 @@ def get_health(self, url: str):
             'url': url
         })
 
+    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
+    # https://github.com/iotaledger/iota-sdk/issues/1921
+    def get_info(self) -> NodeInfoWrapper:
+        """Returns general information about the node together with its URL.
+        GET /api/core/v3/info
+        """
+        return NodeInfoWrapper.from_dict(self._call_method('getInfo'))
+
+    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
```

Comment:
What's different here?

Reply:
where do you see this route in the TIP?

Reply:
It's the third one https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/tip48/tips/TIP-0048/openapi3-core.yaml ?

Reply:
* for `get_info` the parameters (None) match but the return type doesn't.
* for `get_node_info` the parameters don't match, but the return type does.

currently the strict implementation of that spec would be a:
```py
def get_info(self) -> NodeInfo
```
hence the comment.

Reply:
I don't think we need to be that strict, we were at the beginning, but for usability this wasn't satisfying, which is why we changed it to the way it is now
I also think it's fine to keep in this file, since it's still returning the response from this core API endpoint :shrug:

Reply:
Anyways, as for the other I just think the TODO is misleading, but if it's removed in https://github.com/iotaledger/iota-sdk/pull/1928 I'm fine with keeping it now